stable/kong/templates/deployment.yaml

```diff
@@ -224,6 +226,9 @@ spec:
           protocol: TCP
         {{- end }}
         {{- end }}
+        volumeMounts:
```

Comment:
In this container, please expose `9542` and call that port `metrics` so that Prometheus can automatically scrape this endpoint.

Reply:
@hba

done removing server block from controller and adding metrics port to Kong pods.

Shall the podAnnotations: default to prometheus annotation then ?

Reply:
Exposing the port should be enough for some installations while some other installations might need the annotation.
Since pod annotations are already configurable, the user can add those if required.

This is something we can add those too later but I'm being cautious and doing this change incrementally.

Reply:
ok good point. Since all pods would have the prometheus metrics I would have thought having the the prometheus annotations too would make sense. 

Right ok - let me know if there's anything else missing then.